src/Catty/Defines/LanguageTranslationDefines.h

```diff
@@ -452,6 +452,7 @@
 #define kLocalizedSetVolumeTo NSLocalizedString(@"Set volume to", nil)
 #define kLocalizedChangeVolumeBy NSLocalizedString(@"Change volume by", nil)
 #define kLocalizedSetInstrumentTo NSLocalizedString(@"Set Instrument to", nil)
+#define kLocalizedSetTempoTo NSLocalizedString(@"Set Tempo to", nil)
```

Comment:
To reuse translations from Crowdin, it is important to have the exact same language strings as Catroid. Therefore please change to "Set tempo to" with a lower case "t".

Reply:
Thanks for your changes. However, please build the project once more so that the string is also corrected in the other files.